### DISCUSSION TITLE
Add a menu context to search specific strings

### DISCUSSION THREAD

**Discussion by PeterYangTW** [microsoft/vscode-discussions#437] [Extension Development QnA] [answered]
I want to add a menu context option to search specific query strings.
This query string is concatenated by constant string (with regex) and selected string.

For example :
The string = "HANDLE"
When I select "EVENT" in editor and right click menu and click option
It will jump to search viewlet and perform searching "HANDLE*EVENT" automatically.

Here is my code:
```
"contributes": {
    "commands": [
      {
        "command": "testext.hello",
        "title": "HELLO"
      }
    ],
    "menus": {
      "editor/context": [
        {
          "when": "editorTextFocus",
          "command": "workbench.action.findInFiles",
          "args": {
            "query": "HANDLE*${selectedText}",
            "regexp": true
          },
          "group": "navigation"
        }
      ]
    }
```

The option has add to menu succesfully and click it will jump to search viewlet.
But query string has no pass to search viewlet.

(My code is reference from [#95582](https://github.com/microsoft/vscode/issues/95582) and [#97823](https://github.com/microsoft/vscode/issues/97823).)

I want to know How to pass the query string correctly?Thanks

**Comment by isidorn** [comment]
Is this still an issue with the latest version of VS Code Insiders?

**Reply by PeterYangTW** [reply, marked as answer]
I have reported this issue [here](https://github.com/microsoft/vscode/issues/173653) and should be fixed.

**Reply by isidorn** [reply]
Thanks. So I am closing this one as asnwered and fixed.
